packages/flutter/lib/fix_data.yaml

```diff
@@ -17,16 +17,38 @@
 
 version: 1
 transforms:
-  # Changes made in https://github.com/flutter/flutter/pull/110162
-  - title: "Migrate to 'ColorScheme.background'"
-    date: 2022-08-24
+  # Changes made in https://github.com/flutter/flutter/pull/111080
```

Comment:
You'll want to leverage the power of Dart fix to not just remove ThemeData.bottomAppBarColor, but take its value and move it to a component theme. You can use "Migrate to 'TextSelectionThemeData'" as an example, though this will be much simpler since it's only moving 1 color instead of 3.

Reply:
Gotcha. Thanks!